client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -188,6 +216,13 @@ public void onFailure(Exception e) {
                 }
             });
             // end::create-index-execute-async
+
+            assertBusy(() -> {
+                // TODO Use Indices Exist API instead once it exists
```

Comment:
Can we CountDownLatch instead of assertBusy

Reply:
I don't think we can because the CountDownLatch will appear in the documentation (everything between `// tag::foo` and `// end::foo` is copied in the documentation)

Reply:
Ah, ok.